### ISSUE TITLE
Audio crackling/dropout (or micro-freezing game?) in Legend of Kyrandia 1

### ISSUE THREAD

**Issue by mrmatteastwood** [diablodiab/libretro-scummvm-backend#27]
_I mistakenly reported this issue on your other ScummVM repo at first: https://github.com/diablodiab/scummvm/issues/12. Now moving it here._

**Steps to Reproduce:**
1. Get latest ScummVM build from here: https://github.com/hunterk/libretro_builds
2. Get latest ScummVM system files from here: https://github.com/hunterk/libretro_builds/raw/master/scummvm_system_files.zip
3. Run LoK1 on latest RA (1.12.0)

**Current Results:**
There are very noticeable audio dropouts, crackling audio and what appears to be micro-freezing during the game's intro (first minute or so) and also, to a lesser extent, later in the intro and during gameplay. Video: https://youtu.be/Esje0_IlCAs

**Expected Results:**
The game should run without issues.

**Notes:**
- @hizzlekizzle suggests increasing the audio buffer here: https://github.com/diablodiab/libretro-scummvm-backend/blob/main/libretro.cpp#L474. See his post here: https://github.com/libretro/scummvm/issues/179#issuecomment-1288315200
- Audio issues are worst during the first minute or two, but dropouts still occur later in the game as well, e.g. in the scene with Kallak and Malcolm.
- Encountered on my two PCs running Linux Mint 21 and Windows 10.
- Issues also occur when I turn off the fancy shader seen in the video.
- You can see from the shader's noise filter that the game actually micro-freezes, at least at some points.

**Comment by mrmatteastwood** [comment]
I think this is the same issue that @schmurtzm reported on Sep 20, https://github.com/diablodiab/libretro-scummvm-backend/issues/20.

**Comment by i30817** [comment]
Yeah, i previously mentioned i couldn't notice this, but it recently has got much worse for me, and i can see it. I suppose this might be because of the software rendering. The pauses seem to occur whenever the game attempts to change something in the intro, like showing the logo, removing the logo, etc.
